src/Controls/Remote/RemoteVideoMute.tsx

```diff
@@ -26,12 +31,12 @@ const RemoteVideoMute: React.FC<RemoteVideoMuteInterface> = (props) => {
       onPress={() => {
         RtcEngine.muteRemoteVideoStream(
           props.user.uid as number,
-          props.user.video,
+          props.user.video === ToggleState.enabled, //If enabled, disable or vice-versa
         );
-        (dispatch as DispatchType<'UserMuteRemoteVideo'>)({
-          type: 'UserMuteRemoteVideo',
-          value: [props.user, props.user.video],
-        });
+        // dispatch({
```

Comment:
Do we need to update the state from the click event? The SDK sends an event that already mutates the state (RemoteVideoStateChanged).